tools/msPurity/averageFragSpectra.xml

```diff
@@ -41,11 +32,11 @@
         </param>
 
 
-        <param name="snr" type="float" value="0.0" label="Signal-to-noise threshold after averaging or summing" help="" />
+        <param name="snr" type="float" min="0.0" value="0.0" label="Signal-to-noise threshold after averaging or summing" help="" />
```

Comment:
I don't know for this precise parameter. 
But if some option can be found in your underlying R package, you should use `argument="snr"` instead of `name="snr"`. That will display the argument name in the help. It's easier if a user want to search map the Galaxy option and the R manual.
Maybe averageFragSpectra haven't any equivalent in R but maybe some other tools ca

Reply:
Thanks @lecorguille. This is updated now - see latest changes